bbschema/entity.py

```diff
@@ -343,10 +343,11 @@ def update(self, identifier_json):
 
         if 'value' in identifier_json:
             new.value = identifier_json['value']
-        if 'identifier_type_id' in identifier_json:
+
+        if 'identifier_type' in identifier_json:
             new.identifier_type_id =\
-                identifier_json.get('identifier_type', {}).\
-                get('identifier_type_id')
+                identifier_json.get('identifier_type').\
```

Comment:
This won't work - get('identifier_type') will either return an IdentifierType or None. None has no attribute "get", so it'll raise an exception in those cases. Should just be left as is, I think.